### PR TITLE
bonsu: new recipe

### PR DESCRIPTION
**Pull request by tschoonj**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by tschoonj** [comment]
Any thoughts how I can fix the problem on Linux? With yum requirements?

File: recipes/bonsu/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    - bonsu.sequences
+
+about:
+  home: https://github.com/bonsudev/bonsu
+  license: GPL3
+  license_family: GPL3
+  license_file: bonsu/licence/gpl.txt
+  summary: Bonsu - The Interactive Phase Retrieval Suite
+  doc_url: 
+  dev_url:
```
Comment (scopatz) [inline recipes/bonsu/meta.yaml]: Please remove if empty

**Comment by scopatz** [comment]
There are a bunch of CDTs for this package: https://anaconda.org/search?q=libXxf86vm

You'll probably need to add something to the yum requirements too

**Comment by tschoonj** [comment]
I fixed the Linux problem but the Windows one remains... Any thoughts?

**Comment by scopatz** [comment]
windows doesn't have the fftw package available: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=51839&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&taskId=23010c01-6a9d-5f94-9037-792618644b77&lineStart=158&lineEnd=159&colStart=1&colEnd=1

**Comment by tschoonj** [comment]
Can I get some guidance with all this MPI stuff? I have tried following the official [guidelines](https://conda-forge.org/docs/maintainer/knowledge_base.html#complete-example), but it doesn't seem to work...

**Comment by scopatz** [comment]
CC @conda-forge/staged-recipes

File: recipes/bonsu/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    - fftw
+    - fftw * mpi_{{ mpi }}_*  # [mpi != 'nompi']
+  run:
+    - python
+    - pillow
+    - wxpython
+    - vtk
+    - numpy
+    - {{ mpi }}  # [mpi != 'nompi']
+    - fftw
```
Comment (beckermr) [inline recipes/bonsu/meta.yaml]: So it appears the docs don't have this `fftw` in the runtime env. Maybe remove it?

Comment (beckermr) [inline recipes/bonsu/meta.yaml]: So it appears the docs don't have this `fftw` in the runtime env. Maybe remove it?

File: recipes/bonsu/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  host:
+    - pip
+    - setuptools
+    - python
+    - pillow
+    - wxpython
+    - vtk
+    - numpy
+    - {{ mpi }}  # [mpi != 'nompi']
+    - fftw
```
Comment (beckermr) [inline recipes/bonsu/meta.yaml]: I'm also not sure why this one should be there. I am wondering if this is a bug in the docs.

File: recipes/bonsu/meta.yaml (hunk 2 above)
Comment (beckermr) [reply recipes/bonsu/meta.yaml]: Or actually, add a selector (e.g., `ffwt  # [mpi == 'nompi']`

File: recipes/bonsu/meta.yaml (hunk 3 above)
Comment (beckermr) [reply recipes/bonsu/meta.yaml]: Actually, you might add a selector like above here too.

**Comment by beckermr** [comment]
Looking more closely this feels like a conda-build bug.

**Comment by isuruf** [comment]
conda-build 3.18 series has this bug. https://github.com/conda/conda-build/issues/3617

**Comment by beckermr** [comment]
One of the envs on linux has only openmpi in it, but then the solver fails:

```
ValueError: Incompatible component merge:
  - 'mpi_mpich_*'
  - 'mpi_openmpi_*'
```

which is super weird.

**Comment by tschoonj** [comment]
So since this is a bug in `conda` itself, would it be ok if I just dropped Windows support for now?

**Comment by xhochy** [comment]
> So since this is a bug in `conda` itself, would it be ok if I just dropped Windows support for now?

Yes, just drop Windows support here. You can get it back in the feedstock by temporarily pinning the conda-build version there. Feel free to ping me then when you have questions.

**Comment by tschoonj** [comment]
Ok, everything went green now...

**Comment by tschoonj** [comment]
Thanks!

**Pull request merged**